lib/dom.js

```diff
@@ -395,6 +395,7 @@ declare class Element extends Node {
     scrollWidth: number;
     tagName: string;
 
+    closest(): void;
```

Comment:
experimental basically just tells you it is fairly new. But as the DOM Spec is a living standard it won't be removed again.
It is also implemented by the major Browser now. (Latest addition was Safari with Safari 9)